org.mwc.debrief.lite/build.xml

```diff
@@ -119,8 +120,8 @@
 	<target name="clean" depends="quiet,init" description="Clean the plug-in: org.mwc.debrief.lite of all the zips, jars and logs created.">
 		<delete file="${build.result.folder}/debrief_lite.jar"/>
 		<delete file="debrief_lite.jar"/>
-		<delete file="${plugin.destination}/org.mwc.debrief.lite_1.0.377.jar"/>
-		<delete file="${plugin.destination}/org.mwc.debrief.lite_1.0.377.zip"/>
+		<delete file="${plugin.destination}/org.mwc.debrief.lite_${product.version}.jar"/>

```

Comment:
@ayeshare - won't this try to delete any files with the current version?  It's more likely that we will have files that are for the previous product version.  I think we need a wildcard in this delete statement, to allow for all product versions,